plugins/inventory-backend/src/inventory/types.ts

```diff
@@ -14,11 +14,35 @@
  * limitations under the License.
  */
 
+import * as yup from 'yup';
```

Comment:
What is `yup`?

Reply:
A common validation library, to check that inputs/outputs conform to a schema.

There are competing technologies and we aren't enforcing anything at this point, i just picked a common one. This is the one that formik recommends for example. Other suggestions are welcome!